install/0000_00_cluster-version-operator_01_admingate_configmap.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: ConfigMap
+data:
+  ack-4.8-kube-122-api-removals-in-4.9: |
+      Kubernetes 1.22 and thus OpenShift 4.9 remove several APIs which require admin consideration. Please see
+      https://docs.openshift.com/container-platform/4.8/release_notes/ocp-4-8-release-notes.html#ocp-4-8-deprecated-removed-features.
```

Comment:
i hope we get to the point where we can point to better documentation for what an admin needs to go investigate/check before we ship this.

we should probably not merge this PR until we have that in place, since this gate will immediately start showing up in 4.8.z clusters.